Code/Framework/AzCore/AzCore/Math/Aabb.inl

```diff
@@ -59,17 +59,25 @@ namespace AZ
         return CreateCenterHalfExtents(center, Vector3(radius));
     }
 
-
     AZ_MATH_INLINE Aabb Aabb::CreatePoints(const Vector3* pts, size_t numPts)
     {
-        Aabb aabb = Aabb::CreateFromPoint(pts[0]);
```

Comment:
this is assuming the numPts is always greater then 0. if the numPts is zero then the Aabb is null instead of crashing because its trying to access the first element.